### DISCUSSION TITLE
Pimcore 10 - multi-site code organization

### DISCUSSION THREAD

**Discussion by jms-gsulcer** [pimcore/pimcore#8976] [Q&A / Support]
We have a Pimcore instance (v6.5.3) with 14 individual sites in it. Checking out the Pimcore 10 betas, I see that setting Static Routes Controller Action, and setting Page Controller Action, is now a single drop-down, rather than 3 drop-drowns for Bundle, Controller, Action.

What is the recommended way to organize our code-base to make the single drop-down useful? We had each site's code/templates under individual bundles, and the Controller and Action drop-downs "filtered" based on what was selected in the prior dropdowns.

As-is, with a single drop-down, we will have to scroll through over 100 items to find the specific Action for a Page or Static Route.

**Comment by dpfaffenbauer** [comment]
You can still use bundles to organize stuff. Nothing changes so far for you. Only the routes get configured differently

**Reply by jms-gsulcer** [reply]
Ok, thank you.

I was thinking from a UX perspective, configuring the static routes will be a dropdown of almost 200 items to scroll through.  14 sites, each with 5-6 controllers, each controller with 2-3 actions.

I was hoping maybe I had missed a way to organize our code to make these configurations simpler.

**Reply by dpfaffenbauer** [reply]
Use Symfony routes instead, or configure static routes directly in the Config File. But yes, that might be challenging then...
